core/src/validator.rs

```diff
@@ -425,6 +425,11 @@ impl Validator {
             accounts_update_notifier.is_some()
         );
 
+        let system_monitor_service = Some(SystemMonitorService::new(
```

Comment:
Could probably start it first thing in this method.  Maybe add a comment not to start anything before it

Reply:
I don't see anything prior to this where it seems like it would be much more useful. We can always move it up further. It is just logging stats.